src/tests/topp/ProteomicsLFQ_1_subset_out.csv

```diff
@@ -2,17 +2,17 @@ ProteinName,PeptideSequence,PrecursorCharge,FragmentIon,ProductCharge,IsotopeLab
 P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,BSA2,Rep1,2,2,8.861572e07,"BSA2_F1.mzML"
 P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,BSA3,Rep1,4,2,4.999316e07,"BSA3_F1.mzML"
 P02769|ALBU_BOVIN,AEFVEVTK,2,NA,0,L,BSA1,Rep1,5,3,1.546597e08,"BSA1_F2.mzML"
-tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,BSA2,Rep1,2,2,6914.940918,"BSA2_F1.mzML"
-tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,BSA1,Rep1,5,3,7877.708008,"BSA1_F2.mzML"
+tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,BSA2,Rep1,2,2,2.484104e04,"BSA2_F1.mzML"
+tr|A9G0B6|A9G0B6_SORC5,AGAFSLPK,2,NA,0,L,BSA1,Rep1,5,3,3941.330811,"BSA1_F2.mzML"
```

Comment:
substantially different results.
Which one is better? Can this be visually confirmed in the raw data?

Reply:
I would probably need some help here, e.g., running it on the UPS data with @jpfeuffer script.

I wonder why all the current approaches IonQuant, sage, FlashLFQ do no fit at all and just do some simple hill climbing and still (apparently) get pretty good results?

@cbielow @jcharkow @hendrikweisser do you know how much worse FFId would perform (without external seeds) if we would just take the OpenSWATH raw_intensities?

Reply:
One could expose the parameters to Plfq and then nextflow and run a simple for loop including the plotting script. Then we would see.

Reply:
I maybe have time next week. But this can also be a sprint task or a student project.

Reply:
if you do it can you document how you run it and get the plot - so we can give this out as a sprint task next time?

Reply:
Yes. Actually you don't need nextflow. Just download the input from Yassets run and loop over plfq+eval script.
Everything before plfq will not change

Reply:
> do you know how much worse FFId would perform (without external seeds) if we would just take the OpenSWATH raw_intensities?

Sorry, I don't remember the details. In the paper (https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5547443/) we just say that it "helps to increase the robustness of our feature intensity estimation".

I'd be interested in the outcome of your experiment if you test it more thoroughly. Note that results may depend on the quality of the data - I would expect more impact from the elution model fitting for lower-quality raw data (where peak shapes "fluctuate" more).